.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+workflows:
```

Comment:
When I reset configuration, it appears my Engine is left in place. When I then try to recreate the engine again, I receive an error message because that Engine already exists with that name.

Since it's not possible to re-use existing engines, should we just remove the old engine when we Reset Configuration?

Reply:
Have setup a new mechanism to delete existing engine with confirmation popup (6743c8c).
The language selection is now possible when reusing an existing engine since we delete the old one.

You can test it now.

Reply:
Looks great!